stepper.md

```diff
@@ -1,64 +1,214 @@
+17 END_SYSEX                                  (0xF7)
+```
+
+**Stepper zero**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop command                            (0x01)
+3  device number                           (0-9)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper step**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  step command                            (0x02)
+3  device number                           (0-9)
+4  direction                               (0-1) (0x00 = CW, 0x01 = CCW)
 5  num steps byte1 LSB
 6  num steps byte2
-7  num steps byte3 MSB  (21 bits (2,097,151 steps max))
-8  speed LSB            (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
-9  speed MSB
-10 [optional] accel LSB (acceleration in 0.01*rad/sec^2 (1000 = 10.0 rad/sec^2))
+7  num steps byte3 MSB                     (21 bits (2,097,151 steps max))
+8  num speed LSB                           (steps per second)
+9  num speed MSB
+10 [optional] accel LSB                    (acceleration in steps/sec^2)
 11 [optional] accel MSB
-12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
+12 [optional] decel LSB                    (deceleration in steps/sec^2)
 13 [optional] decel MSB
-14 END_SYSEX            (0xF7)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper to**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  to command                              (0x03)
+3  device number                           (0-9)
+4  num target byte1 LSB
+5  num target byte2 MSB
+6  speed LSB                               (steps per second)
+7  speed MSB
+8  [optional] accel LSB                    (acceleration in steps/sec^2)
+9  [optional] accel MSB
+10 [optional] decel LSB                    (deceleration in steps/sec^2)
+11 [optional] decel MSB
+12 END_SYSEX                               (0xF7)
+```
+
+**Stepper enable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  enable command                          (0x04)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper disable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  disable command                         (0x05)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper stop**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop command                            (0x06)
+3  device number                           (0-9)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper reply (sent when a move completes or stop is called)**
```

Comment:
Rename to something more specific such as `Stepper report position` or `Stepper end sequence reply`since there could be other types of replies in the future such as `Stepper getPins`, etc.